format/index.md

```diff
@@ -214,24 +219,18 @@ Here's how an article (i.e. a resource of type "articles") might appear in a doc
 
 #### Attributes <a href="#document-structure-resource-object-attributes" id="document-structure-resource-object-attributes"></a>
 
-A resource object **MAY** contain additional top-level members. These members
-represent "[attributes]" and may contain any valid JSON value.
+A resource object **MAY** contain an "[attributes]" object. If included, this
+object **MUST** reserve the `id` member at it's top level, but can otherwise
+contain any valid JSON value.
```

Comment:
I believe that `type` MUST be reserved as well as `id`. Otherwise, requests to sort or filter by `type` would be ambiguous.

Also: s/it's/its/